ansible/roles/ansible-horizon-plugin/tasks/post_install.yml

```diff
@@ -27,49 +27,73 @@
   shell: "echo {{ PACKAGE_LIB }} > {{ TRILIO_FILE }}"
   when: ENV_PATH != "/usr"
 
-- block:
-  - name: copy tvault_panel_group.py file to remote location
-    copy: src="{{automation_home}}/files/tvault_panel_group.py" dest="{{HORIZON_PATH}}/openstack_dashboard/local/enabled/tvault_panel_group.py"
-    notify:
-      - restart webserver
+##Centos
 
-  - name: copy tvault_admin_panel_group.py file to remote location
-    copy: src="{{automation_home}}/files/tvault_admin_panel_group.py" dest="{{HORIZON_PATH}}/openstack_dashboard/local/enabled/tvault_admin_panel_group.py"
+- block:
+  - name: Find panel files
+    find: 
+      paths: /usr/lib/python3.6/site-packages/dashboards/local/enabled
```

Comment:
can we fetch the python path dynamically?
If the default python version gets upgraded then this path won't be valid